configs/common/packages.yaml

```diff
@@ -141,7 +141,7 @@ packages:
   hdf:
     require: '@4.2.15 +external-xdr ~fortran ~netcdf'
   hdf5:
-    require: '@1.14.3 +hl +fortran +mpi +threadsafe ~szip'
+    require: '@1.14.5 +hl +fortran +mpi +threadsafe ~szip'
```

Comment:
I think this is ok. We need to make sure that all applications test with this prior to rolling out spack-stack@2